deploy/environments/applications.yaml

```
@@ -40,8 +40,8 @@ spack:
     - neurodamus-thalamus+coreneuron%intel^coreneuron+caliper
     - neurodamus-thalamus+coreneuron%intel^coreneuron+caliper+knl
     - parquet-converters
-    - placement-algorithm
-    - py-region-grower
+    - placement-algorithm@2.2.0
+    - placement-algorithm@2.1.4
```

It should be minor if the version enforcing is temporary, but what if we keep the last version unspecified (so spack will use the last one), and force only the old one?
Example for `placement-algorithm`:
```suggestion
    - placement-algorithm
    - placement-algorithm@2.1.4
```

---

thank you! I didn't know about this